### PR TITLE
Landing page content and ES translation updates

### PR DESCRIPTION
**Pull request by Jkrzy**
Landing page updates: [Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/550)

ES translation updates: [Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/563)

## What does this change?
Updated copy for the landing page.
Updated ES translations throughout. 

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/static/sass/custom/landing.scss [hunk 1]
```diff
@@ -169,6 +169,9 @@ $arrow-height: 24px;
       background: color($theme-color-warning);
       color: black !important;
     }
+    a, a:visited {
+      color: white;
```
Comment (cantsin) [inline crt_portal/static/sass/custom/landing.scss:173]: curious why this change?

File: crt_portal/cts_forms/templates/landing.html [hunk 2]
```diff
@@ -170,11 +174,9 @@ <h2 class="h1__display text__reverse" id="your-rights">{% trans "Understanding y
     <div class="grid-container">
       <div class="grid-row grid-gap">
         <div class="tablet:grid-col-12">
-          <h2 class="h2__display">{% trans "Common civil rights violations" %}</h2>
-          <div class="crt-landing--separator_small"></div>
           <div class="grid-row grid-gap">
             <div class="tablet:grid-col-7">
-              <p class="crt-landing--largetext">{% trans "Choose from this list to see examples." %}</p>
+              <h2 class="h2__display f">{% trans "Choose from this list to see example civil rights violations:" %}</h2>
```
Comment (cantsin) [inline crt_portal/cts_forms/templates/landing.html:179]: stray `f` here?

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -703,7 +702,7 @@ msgstr "Delaware"
 
 #: model_variables.py:251
 msgid "District of Columbia"
-msgstr "Distrito de Columbia"
+msgstr "District of Columbia"
```
Comment (cantsin) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:705]: double checking that these reversions to English were wanted?

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -2614,6 +2627,25 @@ msgstr "Revise su informe"
...
+#~ "We enforce federal laws that protect you from discrimination based on "
+#~ "your race, color, national origin, disability status, sex, religion, "
+#~ "familial status, and more."
+#~ msgstr ""
+#~ "Nosotros hacemos cumplir las leyes federales que protegen a las personas "
+#~ "de la discriminación por motivos de su raza, color de piel, origen "
+#~ "nacional, discapacidad, género, religión o estatus familiar."
+
+#~ msgid "Common civil rights violations"
+#~ msgstr "Vulneraciones de derechos civiles comunes"
```
Comment (cantsin) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:2647]: should we remove instead of commenting out?

File: crt_portal/static/sass/custom/landing.scss (hunk 1 above)
Comment (Jkrzy) [reply crt_portal/static/sass/custom/landing.scss:173]: For the mobile background, to match the rest of the text which is white.

File: crt_portal/cts_forms/templates/landing.html (hunk 2 above)
Comment (Jkrzy) [reply crt_portal/cts_forms/templates/landing.html:179]: Oops! Thanks for catching this.
```suggestion
              <h2 class="h2__display">{% trans "Choose from this list to see example civil rights violations:" %}</h2>
```

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po (hunk 3 above)
Comment (Jkrzy) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:705]: From the updated translation spreadsheet I believe so. Can confirm at stand-up

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po (hunk 4 above)
Comment (Jkrzy) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:2647]: Honestly not sure, this is the default behavior of `makemessages` when a translated string is removed from then code, `makemessages` comments it out in the `Django.po` file as opposed to deleting it.  Thoughts?

Comment (cantsin) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:2647]: Totally fine with default behavior. Less merge conflicts down the road if we leave vs delete, I suspect. Thanks for explaining!

**Comment by AvivaOskow** [comment]
@Jkrzy @cantsin @hbillings here are some quick issues I'm seeing, let me know if there is a better way to capture:

# 1 hero image overlap
- [ ] Now that we added the extra line under the button, it is overlapping with the image. Can we make the set height of the image taller now to accommodate? I forget what we set it too @hbillings @cantsin when we were fixing the image before, but you can see the difference in Joe's screenshot above and the mock. This would be a good time to maybe sneak in [this issue](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/572) also, since it involved the blue background behind/above the image.

**JOE'S SCREENSHOT**
Screen Shot 2020-06-11 at 12 32 42 PM

**SHOULD BE*
Screen Shot 2020-06-11 at 12 34 59 PM

# "About the Division" columns
- [ ] Are we using the correct grid columns for the About the Division section? The first subhead under the left column "Enforcement" shouldn't need to break into 4 lines if so. When I tried this in the inspector before it fit on 3 lines (and in Sketch but I know that can be different. This might be the type normalization issue @hbillings? @cantsin )

_I don't want to spend too much time on this, I just know at one point we weren't using the grid for these columns and instead were using spacing between them, so I wanted to check_

**JOE'S SCREENSHOT**
Screen Shot 2020-06-11 at 12 38 25 PM

**GRID EXAMPLE**
Screen Shot 2020-06-11 at 12 42 52 PM

# Protected characteristics list
- [ ] Looks like the Protected Characteristics column got shifted up when we removed the subhead for the examples section. That section should still align with the top of the examples dropdown.

**JOE'S SCREENSHOT**
Screen Shot 2020-06-11 at 12 46 27 PM

**Should Be**
Screen Shot 2020-06-11 at 12 47 21 PM

**Comment by AvivaOskow** [comment]
I put these in order of the page, but in terms of priority, I would say
- hero image spacing fix
- Protected characteristics alignment fix
- See if the columns are correct for "About the division" section
